src/requests/getImages.js

```diff
@@ -0,0 +1,26 @@
+import axios from 'axios';
+
+const getImages = query => {
+	if (!query) {
+		return Promise.resolve([]);
+	} else {
+		return axios
+			.get(`https://images-api.nasa.gov/search?q=${query}`)
+			.then(response => {
+				const imageResults = response.data.collection.items;
+
+				const parsedImages = imageResults.filter(
+					image => image.data[0].media_type === 'image',
+				);
+
+				const images = parsedImages.map(image => image.links[0].href);
+
+				return images;
```

Comment:
Could be simplified a little:
```suggestion
				return parsedImages.map(image => image.links[0].href);
```